src/core/accido/_class_adjective.py

```
@@ -5,6 +5,7 @@
 import logging
 from functools import total_ordering
 from typing import TYPE_CHECKING, Literal, overload
+from warnings import deprecated
```

Pylint (3.3.7)

[error] No name 'deprecated' in module 'warnings'

(E0611)